src/core/1/guides/essentials/cli/index.md

```diff
@@ -418,3 +426,164 @@ The roles, profiles and users definition follow the same structure as in the bod
+
+**Notes:**
+
+* The mapping can contain any number of index and collection configurations.
+* Field definitions follow the [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/mapping.html) mapping format.
+* If an index or collection does not exist, it will be created automatically.
+* Mappings are loaded sequentially, one index/collection pair at a time. If a failure occurs, Kuzzle immediately interrupts the sequence.
+
+---
+
+## loadFixtures
+
+{{{since "1.6.6"}}}
+
+```bash
+./bin/kuzzle loadFixtures <file>
+
+# [✔] Fixtures have been successfully loaded
+```
+
+The `loadFixtures` command load fixtures directly into the storage layer.
+
+### Fixtures file example
+
+```js
+{
+  "index-name": {
+    "collection-name": [
+      {"create": { "_id": "uniq-id-123456" }},
+      {"field": "value", "field2": "value", "field...", "value"}
+    ]
+  }
+}
+```
+
+**Notes:**
+
+* The fixtures can contain any number of index and collection configurations.
+* Each collection contains an array of data to load, just like the [bulk:import API]({{ site_base_path }}api/1/controller-bulk/import/).
+* If an index or collection does not exist, the load will fail.
+* Fixtures are loaded sequentially, one index/collection pair at a time. If a failure occurs, Kuzzle immediately interrupts the sequence.
+
+---
+
+## loadSecurities
+
+{{{since "1.6.6"}}}
+
+```bash
+./bin/kuzzle loadSecurities <file>
+
+# [✔] Securities have been successfully loaded
+```
+
+The `loadSecurities` command load roles, profiles and users directly into the storage layer.
+
+The roles, profiles and users definition follow the same structure as in the body parameter of the API:
+
+ - [createRole]({{ site_base_path }}api/1/controller-security/create-role)
+ - [createProfile]({{ site_base_path }}api/1/controller-security/create-profile)
+ - [createUser]({{ site_base_path }}api/1/controller-security/create-user)
+
+### Securities file example
+
+```js
+{
+  "roles": {
+    "role-id": {
+      /* role definition */
+    }
+  },
+  "profiles": {
+    "profile-id": {
+      /* profile definition */
+    }
+  },
+  "users": {
+    "user-id": {
+      /* user definition */
+    }
+  }
+}
+```
+
+**Notes:**
+
+* The file can contain any number of roles, profiles and users.
+* If a role, profile or user already exists, it will be replaced.
+* Fixtures are loaded sequentially, first the roles, then the profiles and finally the users. If a failure occurs, Kuzzle immediately interrupts the sequence.
+
+## encryptSecrets
+
+{{{since "1.8.0"}}}
+
+```bash
+./bin/kuzzle encryptSecrets [file] [options]
+
+Options:
+      --vault-key <vaultKey>     Vault key used to encrypt secrets
+      --outputFile <outputFile>  Output file to write encrypted secrets
+      --noint                    non interactive mode
+```
+
+Encrypt the provided `file` with the provided `vaultKey`.  
+The `vaultKey` can be either provided in the command line or in the `KUZZLE_VAULT_KEY` environment variable.  
```

Comment:
Maybe you can precise that it's provided in an environment variable _by default_, but have the possibility to give it as a program parameter (even if it seems obvious)

Reply:
It's not provided by an environment variable by default, by default there is nothing specifying the vault key